runtime/permissions.rs

```diff
@@ -642,7 +678,7 @@ fn permission_prompt(message: &str) -> bool {
     return false;
   };
   let msg = format!(
-    "️{}  {}. Grant? [g/d (g = grant, d = deny)] ",
+    "{}  ️Deno requests {}. Grant? [g/d (g = grant, d = deny)] ",
```

Comment:
Not sure if we want to add `Deno requests`, opinions @ry?

Reply:
This is the current, just that currently everywhere the function is called "Deno requests " is put in the request string. This is just deduplication

Reply:
I see, then LGTM